presto-docs/src/main/sphinx/optimizer/join-distribution-type.rst

```diff
@@ -0,0 +1,33 @@
+********************************
+Partitioned and Replicated Joins
+********************************
+
+Background
+----------
+Presto can perform two types of distributed joins: partitioned and replicated.
+
+In a partitioned join, both inputs to a join get hash partitioned across the nodes of the cluster. This distribution type is good for larger inputs,
+as it requires less memory on each node. However, it can be much slower than replicated joins because it can require more data to be transferred
+over the network.
+
+In a replicated join, the input on the right side is distributed to all of the nodes on the cluster containing data from the input on the left.
```

Comment:
that's only true when we use `join-distribution-type=automatic`.  Otherwise it's always the right table.